Figure out why Travis tests can't run (nvm)

https://travis-ci.org/algolia/algoliasearch-client-javascript/jobs/322857164


```
nvm is not compatible with the "npm_config_prefix" environment variable: currently set to "/home/travis/.nvm/versions/node/v9.3.0"
Run `unset npm_config_prefix` to unset it.
Downloading and installing node v7.10.1...
Downloading https://nodejs.org/dist/v7.10.1/node-v7.10.1-linux-x64.tar.gz...
######################################################################## 100.0%
Computing checksum with sha256sum
Checksums matched!
nvm is not compatible with the "npm_config_prefix" environment variable: currently set to "/home/travis/.nvm/versions/node/v9.3.0"
Run `unset npm_config_prefix` to unset it.
npm ERR! code ELIFECYCLE
npm ERR! errno 11
npm ERR! algoliasearch@3.24.8 [secure]-ci: `./scripts/[secure]-ci`
npm ERR! Exit status 11
npm ERR! 
npm ERR! Failed at the algoliasearch@3.24.8 [secure]-ci script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2017-12-29T10_57_36_400Z-debug.log
```

I looked around and couldn't find any cause in the nvm issues

--- a/scripts/test-node
+++ b/scripts/test-node
@@ -13,6 +13,8 @@ set -e # exit when error
 [ -z $TRAVIS_PULL_REQUEST ] && TRAVIS_PULL_REQUEST='false'
 [ -z $TRAVIS_BUILD_NUMBER ] && TRAVIS_BUILD_NUMBER='false'
 
+# nvm has a special setup on Travis where it might break installing other versions
+unset npm_config_prefix
 nvm install 7
 node test/run-node.js | tap-bail | tap-dot
 

